index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+  <nav class="container">
+    <img id="logo" src="img/image_geometry_1.png" alt="">
+    <div id="bars">
+      <a class="active" href="#"><i class="fa fa-bars" style="color:black;"></i></a>
+    </div>
+  </nav>
+  <header class="heading">
+    <div id="name">
+      Kashif Saleem
+    </div>
+  </header>
+  <intro class="intro">
+    <p id="para">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+  </intro>
+  <div class="button">
+    <a id="btn" href="">Start collaboraton</a>
+  </div>
```

Comment:
- [ ] For Making your HTML markup is well structured and follows a clear hierarchy and semantic correct I recommend you to use `section` tag for headline section.
- [ ] I recommend you to use `h1` tag for main title("Hello! I am a software developer!") and `p` tag for paragraph("I can help you build a product, feature or website...")

Reply:
**Please see if this is correct now**
![Capture](https://user-images.githubusercontent.com/111677605/198226484-e6092cdb-35fc-43e5-b02d-5b07fd556411.JPG)

Reply:
**Please see if this is correct now**
![changed](https://user-images.githubusercontent.com/111677605/198227182-63d83f98-da0a-447c-87eb-4c2277aeaa8f.JPG)

Reply:
Hello @Kashif-Saleem-Ghuman  good for using `section ` tag but still i recommend you to include `button`  inside the `section` tag